src/wazuh_modules/vulnerability_scanner/src/databaseFeedManager/updateCVEDescription.hpp

```diff
@@ -0,0 +1,130 @@
+/*
+ * Wazuh Vulnerability scanner - Database Feed Manager
+ * Copyright (C) 2015, Wazuh Inc.
+ * Oct 3, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#ifndef _UPDATE_CVE_DESCRIPTION_HPP
+#define _UPDATE_CVE_DESCRIPTION_HPP
+
+#include "chainOfResponsability.hpp"
+#include "cve5_generated.h"
+#include "eventContext.hpp"
+#include "flatbuffers/flatbuffers.h"
+#include "rocksDBWrapper.hpp"
+#include "vulnerabilityDescription_generated.h"
+
+constexpr auto DESCRIPTION_DATABASE_PATH {"queue/vd/descriptions"};
+constexpr auto CVE5_DATABASE_PATH {"queue/vd/cve5"};
+
+/**
+ * @brief UpdateCVEDescription class.
+ *
+ */
+class UpdateCVEDescription final
+{
+public:
+    /**
+     * @brief Reads CVE5 database, creates a vulnerability description flatbuffer and stores it in a specific RocksDB
+     * database.
+     *
+     * @param resource CVE ID.
+     * @param cve5Flatbuffer CVE5 Flatbuffer.
+     */
+    static void storeVulnerabilityDescription(const std::string& resource, const cve_v5::Entry* cve5Flatbuffer)
+    {
+        auto metricsArray = cve5Flatbuffer->containers()->cna()->metrics();
```

Comment:
We should be completely paranoid in my opinion using flatbuffers, and check every field before using it.

Reply:
As you've already mentioned, these fields should be present because they are mandatory in CVE 5.0

Reply:
I assume that the rocksDB database must have a valid cve5 schema and containers and cna are required for that. But I can add some checks, I think it is the way to go, always expect the worst case.